lib/ansible/modules/user.py

```diff
@@ -560,6 +591,26 @@ def backup_shadow(self):
         if not self.module.check_mode and self.SHADOWFILE:
             return self.module.backup_local(self.SHADOWFILE)
 
+    def set_password_expire_max(self):
+        command_name = 'chage'
+
+        cmd = [self.module.get_bin_path(command_name, True)]
+        cmd.append('-M')
+        cmd.append(self.password_expire_max)
+        cmd.append(self.name)
+
+        return self.execute_command(cmd)
```

Comment:
Rather than running this command every time, we need to check the current value on the system and compare it to the value from the task.

The `user_password()` and `parse_shadow_file()` methods should be extended to return this information, then use that to determine whether or not to run these commands.